builds/e2e/templates/longhaul-setup.yaml

```diff
@@ -6,6 +6,10 @@ parameters:
   quickstart.artifactName: ''
 
 steps:
+  - script: |
+      sudo iotedge system stop || true
+      sudo systemctl stop iotedge || true
+    displayName: Immediately attempt to stop iotedge service (needed for arm)
```

Comment:
Want to call out two things:
1. There is a stop command running in quickstart, which will get called later. But this is too late. We should leave the quickstart stop logic as-is because other non-arm pipelines rely on it (i.e. nested longhaul). And others might in the future.
2. This doesn't need to be added to nested longhaul because nested longhaul is amd only.

Reply:
Is this worth having as a comment?